### PR TITLE
[TASK] Add a CHANGELOG.md

### LINKED ISSUE #50: Add a CHANGELOG.md
http://keepachangelog.com/en/0.3.0/

### PR DESCRIPTION
**Pull request by oliverklee**
This file lists all changes that affect the way the Gem works or how it
can be used, but not non-functional cleanup.

Fixes #50

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Page Title Helper Change Log
+
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+
+## 3.0.0 (unreleased)
```
Comment (lwe) [inline CHANGELOG.md]: Would use `## [Unreleased]` tag

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Page Title Helper Change Log
```
Comment (lwe) [inline CHANGELOG.md]: In the readme it's written _Page title helper_

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+All notable changes to this project will be documented in this file.
```
Comment (lwe) [inline CHANGELOG.md]: Reference somewhere to http://keepachangelog.com/en/0.3.0/ as well or is this smth for CONTRIBUTING file?

File: README.md [hunk 4]
```diff
@@ -3,8 +3,6 @@
 [![Build Status](https://travis-ci.org/lwe/page_title_helper.svg?branch=master)](https://travis-ci.org/lwe/page_title_helper)
 [![Gem Version](https://badge.fury.io/rb/page_title_helper.svg)](http://badge.fury.io/rb/page_title_helper)
 
-This project adheres to [Semantic Versioning](http://semver.org/).
```
Comment (lwe) [inline README.md]: Why not also keep this in the README, this is one of the first things a user reads and sees and I'd say it's valuable in here as duplicate info too.

File: CHANGELOG.md (hunk 2 above)
Comment (oliverklee) [inline CHANGELOG.md]: Changed.

File: CHANGELOG.md (hunk 1 above)
Comment (oliverklee) [inline CHANGELOG.md]: Changed.

File: README.md (hunk 4 above)
Comment (oliverklee) [inline README.md]: Reverted.

File: CHANGELOG.md (hunk 3 above)
Comment (oliverklee) [inline CHANGELOG.md]: added.

**Comment by oliverklee** [comment]
All comments implemented, rebased and repushed.

**Comment by oliverklee** [comment]
@lwe Could you please re-review? Thanks!

**Comment by oliverklee** [comment]
rebased

**Pull request merged**